src/Sylius/Behat/Context/Setup/UserContext.php

```diff
@@ -172,4 +172,22 @@ public function accountWasDeleted($email)
 
         $this->userRepository->remove($user);
     }
+
+    /**
+     * @Given there was account of :email with password :password
+     */
+    public function thereWasAccountOf($email, $password)
+    {
+        $this->thereIsUserIdentifiedBy($email, $password);
```

Comment:
Not sure if we need a separate method for that step. I would add this annotation to `thereIsUserIdentifiedBy` method.